index.bs

```diff
@@ -698,6 +731,8 @@ Note: the support of depth sensing API is not limited only to hardware classifie
 
 For each of {{XRDepthStateInit/depthTypeRequest}}, {{XRDepthStateInit/usagePreference}}, and {{XRDepthStateInit/dataFormatPreference}},  The device MUST have a <dfn>preferred [=native depth sensing=] capability</dfn> that MUST be used if the corresponding array is empty. The type, usage, and format SHOULD reflect the most efficient ones of the device, though they may be dependent upon each other.
 
+The device can be said to have a <dfn>depth sensing active state</dfn>, which is a boolean representing if depth sensing capabilities are actively running. This state should begin as <code>true</code>. If the native device cannot simulate this state, the user agent can simply not process any depth data when the state is otherwise <code>false</code>.
```

Comment:
Not clear on what this last sentence is trying to say?

"If the native device cannot simulate this state, the user agent can simply not process any depth data when the state is otherwise `false`"?

Is that trying to say that the state should be false if the device can't simulate depth? Or that the device is allowed to not process depth when it's false? (If so, that feels like it should be a non normative note, since it's a suggested implementation detail.)

Reply:
Yeah, it was intended to tell the UA that they should do what they can to minimize the cost of having depth data enabled if their runtime can't disable it.

Reply:
Switched to: `If the native device cannot simulate this state, the user agent SHOULD take steps to mitigate the performance impact of this feature being enabled when this state is <code>false</code>.`

Rather than a non-normative note, as I think it conveys the intent better.